app/src/main/java/jp/sample/ocrsampleandroid/MainActivity.kt

```diff
@@ -40,7 +40,7 @@ class MainActivity : AppCompatActivity() {
 
                 val uri = data.data
                 val uri2 = data.data
-                val uri3aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa = data.data
+                val uri3aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa = data.data
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tarumzu/OCRSampleAndroid/blob/d6e748953407126308961bd78a39d1dee050e62e/app/src/main/java/jp/sample/ocrsampleandroid/MainActivity.kt#L43"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>